src/main/java/org/jabref/gui/Base.css

```
@@ -265,6 +265,8 @@
     -jr-ai-message-user-border: -jr-theme;
     -jr-ai-message-ai: -jr-accent;
     -jr-ai-message-ai-border: -jr-theme;
+    -jr-ai-message-error: -jr-light-red;
+    -jr-ai-message-error-border: -jr-red;
```

The border is maybe too contrasty. What about 

```suggestion
    -jr-ai-message-error-border: derive( -jr-ai-message-error, 20%);
```

I don't know how to try out, did not see any error message.